packages/core/src/messagingProtocols/proteus/ProteusService/CryptoClient/CoreCryptoWrapper/PrekeysTracker/PrekeysTracker.store.ts

```diff
@@ -17,49 +17,32 @@
  *
  */
 
-import {CoreDatabase} from '../../../../../../storage/CoreDB';
-
-const STATE_PRIMARY_KEY = 'prekeys_state';
-
 type PrekeysState = {
   nbPrekeys: number;
-  highestId: number;
 };
-export class PrekeysTrackerStore {
-  constructor(private readonly db: CoreDatabase) {}
 
-  private async getState(): Promise<PrekeysState> {
-    return (await this.db.get('prekeys', STATE_PRIMARY_KEY)) ?? {nbPrekeys: 0, highestId: 0};
-  }
+let state: PrekeysState = {nbPrekeys: 0};
```

Comment:
Now the state is an in memory state only. No need for database access anymore (since it will be synced at load time with backend)